packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/managedByOvh/pack-xdsl-modem-managedByOvh.html

```diff
@@ -12,7 +12,7 @@
                     data-ng-disabled="!ManagedByCtrl.mediator.capabilities.canChangeManagement || ManagedBy.updating"
                     data-ng-really-click="ManagedByCtrl.changeManagedBy()"
                     data-ng-really-undo="ManagedByCtrl.undo()"
-                    data-ng-really-message="{{ ManagedByCtrl.mediator.info.managedByOvh ? 'xdsl_modem_managedBy_really_on' : 'xdsl_modem_managedBy_really_off' | translate }}
+                    data-ng-really-message="{{ ManagedByCtrl.mediator.info.managedByOvh ? 'xdsl_modem_managedBy_really_off' : 'xdsl_modem_managedBy_really_on' | translate }}
```

Comment:
@JeremyDec Translations are not changed, just used the correct message with the correct action (when the checkbox is not set, display the message 'xdsl_modem_managedBy_really_on' and not what it was displayed)